services/gitclub/app/routes/role_assignments.py

```diff
@@ -75,19 +102,19 @@ def org_update(org_id):
     user = {"type": "User", "id": payload["username"]}
     if not authorize("read", user):
         raise NotFound
-    oso.bulk_delete(
-        [
-            ["has_role", object_to_typed_id(user), role, object_to_typed_id(org)]
-            for [_, _, role, _] in get("has_role", user, None, org)
-        ]
+
+    bulk_update(
```

Comment:
Look how much more pleasant this became! No need to call `get` or do list comprehension stuff.